j-ngo828/TackleIt
# Setup frontend with typescript react vite

## What does this PR do?

- Setup frontend with TypeScript + SWC React Vite
- Setup Prettier and ESLint 
- Setup alias import

## How should this be tested?

- Run `npm run dev` from `./frontend` and ensure you can access the application at `localhost:5173`
- Ensure Prettier formatter and ESLint linter works properly
